kafka/consume.go

```diff
@@ -74,9 +78,10 @@ func NewConsumer(log logrus.FieldLogger, conf Config, opts ...ConfigOpt) (*Confl
 	}
 
 	confluenceConsumer := &ConfluentConsumer{
-		c:    consumer,
-		conf: conf,
-		log:  log,
+		c:         consumer,
+		conf:      conf,
+		log:       log,
+		partition: -1,
```

Comment:
would not be more idiomatic if the type is `*uint32`? so we can compare with `nil`?

Reply:
`-1` is a idiomatic way look at strings.Index or io.EOF